dna/tests/src/fixtures.rs

```diff
@@ -0,0 +1,181 @@
+#[cfg(test)]
```

Comment:
```suggestion
// in this file, you will see instances of setting up a type alias, such as `type OptionVecString = Option<Vec<String>>;`
// this is done to enable passing those to constructor functions for fixturators, like
// ```
// fixturator!(
//      Goal;
//        constructor fn new(..., OptionVecString, ...);
//  );
// ```
#[cfg(test)]
```